File: Examples/CSharp/AsposePDF/Stamps-Watermarks/ImageinHeader.cs
//////////////////////////////////////////////////////////////////////////
// Copyright 2001-2013 Aspose Pty Ltd. All Rights Reserved.
//
// This file is part of Aspose.Pdf. The source code in this file
// is only intended as a supplement to the documentation, and is provided
// "as is", without warranty of any kind, either expressed or implied.
//////////////////////////////////////////////////////////////////////////
using System.IO;

using Aspose.Pdf;

namespace CSharp.AsposePdf.StampsWatermarks
{
    public class ImageinHeader
    {
        public static void Run()
        {
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_AsposePdf_StampsWatermarks();

            //open document
            Document pdfDocument = new Document(dataDir+ "ImageinHeader.pdf");

            //create header
            ImageStamp imageStamp = new ImageStamp(dataDir+ "aspose-logo.jpg");
            //set properties of the stamp
            imageStamp.TopMargin = 10;
            imageStamp.HorizontalAlignment = HorizontalAlignment.Center;
            imageStamp.VerticalAlignment = VerticalAlignment.Top;
            //add header on all pages
            foreach (Page page in pdfDocument.Pages)
            {
                page.AddStamp(imageStamp);
            }

            //save updated document
            pdfDocument.Save(dataDir+ "ImageinHeader_out.pdf");
            
            
        }
    }
}

File: Examples/VisualBasic/AsposePDF/Stamps-Watermarks/ImageinHeader.vb
Imports System.IO
Imports System
Imports Microsoft.VisualBasic
Imports Aspose.Pdf

Namespace AsposePDF.StampsWatermarks
    Public Class ImageinHeader
        Public Shared Sub Run()
            ' ExStart:ImageinHeader
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_AsposePdf_StampsWatermarks()

            ' Open document
            Dim pdfDocument As New Document(dataDir & Convert.ToString("ImageinHeader.pdf"))

            ' Create header
            Dim imageStamp As New ImageStamp(dataDir & Convert.ToString("aspose-logo.jpg"))
            ' Set properties of the stamp
            imageStamp.TopMargin = 10
            imageStamp.HorizontalAlignment = HorizontalAlignment.Center
            imageStamp.VerticalAlignment = VerticalAlignment.Top
            ' Add header on all pages
            For Each page As Page In pdfDocument.Pages
                page.AddStamp(imageStamp)
            Next
            dataDir = dataDir & Convert.ToString("ImageinHeader_out_.pdf")
            ' Save updated document
            pdfDocument.Save(dataDir)
            ' ExEnd:ImageinHeader
            Console.WriteLine(Convert.ToString(vbLf & "Image in header added successfully." & vbLf & "File saved at ") & dataDir)
        End Sub
    End Class
End Namespace

